nginx/nginx.conf

```diff
@@ -0,0 +1,13 @@
+server {
```

Comment:
Konfiguracja serwera, który będzie uruchamiał build, testowane lokalnie na obrazie dokerowym i działało, nie wiem czy będzie to działało na render z tymi ustawieniami.